Dockerfile

```diff
@@ -5,17 +5,13 @@ WORKDIR /dhx/node
 
 COPY . .
 
-RUN apt-get update && apt-get install -y build-essential wget cmake pkg-config libssl-dev \
+# https://substrate.dev/docs/en/knowledgebase/getting-started
+RUN apt-get update && apt-get install -y build-essential wget cmake pkg-config libssl-dev llvm \
     openssl git gcc clang libclang-dev curl vim unzip screen bash \
     && curl https://getsubstrate.io -sSf | bash -s -- --fast \
-    && wget -O - https://sh.rustup.rs | sh -s -- -y \
+    && . /root/.cargo/env \
```

Comment:
Shouldn't be there no space?

Reply:
i actually haven't tested that this PR works yet.

but that command is equivalent to the command `source ~/.cargo/env` that's shown here https://substrate.dev/docs/en/knowledgebase/getting-started/#manual-rust-configuration
it's just that i've found that `.` always works but sometimes `source` doesn't, so i always used `.`.
it's to apply the changes made to that configuration in the current terminal session.

unfortunately i can't find documentation about using `.` instead of `source`